extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -660,11 +660,10 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
       this.extensions_./*OK*/installExtensionForDoc(
           this.getAmpDoc(), 'amp-analytics');
     }
-
     const refreshInterval = Number(responseHeaders.get('amp-force-refresh'));
-    if (refreshInterval && !getPublisherSpecifiedRefreshInterval(
-        this.element, this.win, 'doubleclick')) {
+    if (refreshInterval) {
       this.element.setAttribute(DATA_ATTR_NAME, refreshInterval);
+      this.initRefreshManagerIfEligible_();
```

Comment:
I don't think you need this here given layoutCallback would just trigger refresh naturally.  Also, why did you remove the getPublisherSpecifiedRefreshInterval?